fitbenchmarking/fitbenchmarking
# Adds options test for getboolean and getint

#### Description of Work

Fixes #392 

Adds options test for `getboolean` and `getint` functions in options 

#### Testing Instructions

1. Check test work as expected

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
